.github/workflows/release.yml

```
@@ -0,0 +1,36 @@
+# .github/workflows/publish-release.yml
+name: Publish release
+on:
+  push:
+    tags:
+      - "v*"
+jobs:
+  github_release:
+    name: GitHub release
+    runs-on: ubuntu-latest
```

I would consider to lock version instead of latest tag, which over the years might introduce an unexpected behaviour.
```suggestion
    runs-on: ubuntu-24.04
```

Same for other 'latest' images.